lib/matplotlib/tri/tritools.py

```diff
@@ -12,18 +12,19 @@ class TriAnalyzer:
     """
     Define basic tools for triangular mesh analysis and improvement.
 
-    A TriAnalyzer encapsulates a `~matplotlib.tri.Triangulation` object and
-    provides basic tools for mesh analysis and mesh improvement.
+    A TriAnalyzer encapsulates a `.Triangulation` object and provides basic
+    tools for mesh analysis and mesh improvement.
 
     Attributes
     ----------
     scale_factors
 
     Parameters
     ----------
-    triangulation : `~matplotlib.tri.Triangulation`
+    triangulation : `.Triangulation`
```

Comment:
```suggestion
    triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above